src/teleop/gui/src/components/ERDTask.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+<div>
+    <p>ERD</p>
```

Comment:
Random thing but do we really want to separate ERD and ES? The configurations were the same the previous year so the only difference would've been that the map wasn't being used in ES. I personally think that it would just make it somewhat annoying to maintain two separate pages so we might want to consider changing the vues.

The main difference that I can think of about functionality would be the camera resolutions being dependent on the guis but this can easily be fixed/changed if we change the camera interface.

Reply:
This is definitely something we can discuss. If I remember correctly they were separated fairly late in the season last year. @CameronTressler can probably say a bit more on that I don't remember the exact reasoning as to how it ended up that way.

Reply:
I think we should have two separate pages, but ultimately it's up to Joe. My reasoning is that you can avoid code duplication by using Vue components. The ERD and ES pages just differ in layout, but they use mostly the same components.

Reply:
Fair